.gitignore

```diff
@@ -20,3 +20,6 @@ tmp
 mkmf.log
 target
 ruby/ext/rust_example/Makefile
+ruby/ext/rust_example/rust_example.o
+ruby/ext/rust_example/rust_example.so
+ruby/ext/rust_example/librust_example.a
```

Comment:
That makes sense. Maybe we could move this to `ruby/ext/rust_example/{arch}` so you can copy them if needed?